packages/ra-core/src/dataProvider/useGetList.ts

```diff
@@ -57,7 +67,7 @@ const useGetList = <RecordType extends Record = Record>(
     pagination: PaginationPayload,
     sort: SortPayload,
     filter: object,
-    options?: any
+    options?: UseGetListOptions
```

Comment:
use the existing `UseDataProviderOptions` instead

Reply:
Good point, I did it as well for `useGetOne()`.